.github/workflows/ci.yml

```
@@ -0,0 +1,29 @@
+name: Build
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - '*'
+
+jobs:
+  build:
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+        node-version:
+          - 14.x
+          - 15.x
```

Something like this would be probably more useful: 

```suggestion
          - 12.x
          - 14.x
```

---

I'm fine with this change.
My reasoning was to use the current LTS (14) and the latest stable version (15) but I'm fine with 12.x and 14.x.